docker/seccomp.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+def loadDefault( filename ):
+    with open( filename, 'r' ) as f:
+        return json.load( f )
+
+def inPlaceTransform( jdata ):
+    # for names_action_etc in jdata[ 'syscalls' ]:
+    #     names = names_action_etc[ 'names' ]
+    #     action = names_action_etc[ 'action' ]
+    #     if 'personality' in names:
+    #         print( action )
+    jdata[ 'syscalls' ].extend(
+        { 
+            "names": [ "personality" ],
+            "action": "SCMP_ACT_ALLOW",
+            "args": [ { "index": 0, "value": value, "op": "SCMP_CMP_EQ" } ]
+        }
+        for value in [ 0x0040000, 0x0400000, 0x0440000 ]
+    )
+
+if __name__ == "__main__":
+    jdata = loadDefault( sys.argv[1] )
```

Comment:
All local vars here are in the global namespace. Beware! Never define vars within an `if __name__ == '__main__'` block. Always right a `main` function that defines vars inside.

Reply:
I presume the concern is about using the script as a module rather than a main. Fair point.